integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiClusterResource.java

```diff
@@ -0,0 +1,821 @@
+    }
+
+    kubectlScaleCluster(cluster2Res,domainNamespace,3);
+    checkPodReadyAndServiceExists(managedServer2Prefix + 3, domainUid, domainNamespace);
+    logger.info("Cluster is scaled up to replica count 3");
+
+    // Clean up resources
+    deleteDomainResource(domainUid, domainNamespace);
+    deleteClusterCustomResourceAndVerify(cluster1Res,domainNamespace);
+    deleteClusterCustomResourceAndVerify(cluster2Res,domainNamespace);
+  }
+
+  /**
+   * Create a WebLogic domain DR with domain level replica set to zero.
+   * Create  kubernates cluster resources CR which corresponds to a 
+   * WebLogic cluster cluster-1 in model file.
+   * Associate Cluster Resource CR with Domain Resource DR  
+   * Make sure only managed servers from cluster-1 comes up
+   * Delete the cluster Resource CR
+   * Make sure Domain Validation Error event is generated. 
+   *
+   * message: 'Domain domain2 failed due to 'Domain validation error': 
+   * Cluster resource 'domain2-cluster-1' not found in namespace 
+   * 'ns-ptkhxk'. Update the domain resource to correct the validation error.'
+   *
+   * However the managed servers should not stop.
+   */
+  @Test
+  @DisplayName("Verify removal of a cluster resource generate Error Event")
+  void testDeleteClusterResource() {
+
+    String domainUid = "domain2";
+    String adminServerPodName = domainUid + "-admin-server";
+    String managedServerPrefix = domainUid +  "-c1-managed-server";
+
+    String clusterRes   = domainUid + "-cluster-1";
+    String clusterName  = "cluster-1";
+    String configMapName = domainUid + "-configmap";
+
+    deleteDomainResource(domainUid, domainNamespace);
+    deleteClusterCustomResourceAndVerify(clusterRes,domainNamespace);
+ 
+    // create and deploy cluster resource(s)
+    ClusterResource cluster = createClusterResource(
+        clusterRes, clusterName, domainNamespace, replicaCount);
+    logger.info("Creating ClusterResource {0} in namespace {1}",clusterRes, domainNamespace);
+    createClusterAndVerify(cluster);
+    createModelConfigMap(domainUid,configMapName);
+
+    // create and deploy domain resource
+    DomainResource domain = createDomainResource(domainUid,
+               domainNamespace, adminSecretName,
+        TEST_IMAGES_REPO_SECRET_NAME, encryptionSecretName,
+        MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG, configMapName);
+    logger.info("Creating Domain Resource {0} in namespace {1} using image {2}",
+        domainUid, domainNamespace, 
+        MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG);
+    domain.getSpec().withCluster(new V1LocalObjectReference().name(clusterRes));
+
+    createDomainAndVerify(domain, domainNamespace);
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+    // check managed server pods are not started
```

Comment:
The comment and the code does not match - the comment here says the managed server pods are not started, but the code that followed checks if the managed servers are started.